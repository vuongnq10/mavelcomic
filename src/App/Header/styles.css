.home {
  background: url('./banner.jpg') no-repeat center center; 
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  min-height: 500px;
}

.slogan {
  margin: 30px auto;
  text-align: center;
  max-width: 500px;
  color: #fff;
}

.description {
  margin: 0px auto;
  text-align: center;
  color: #fff;
}