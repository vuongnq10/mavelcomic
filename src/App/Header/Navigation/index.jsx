import React from 'react';
import Navbar from 'react-bootstrap/Navbar';
import Nav from 'react-bootstrap/Nav';

import styles from './styles.css';
import Icon from './favicon.ico';

const Index = () => (<Navbar expand="lg" variant="light">
  <Navbar.Brand href="/">
    <img src={Icon} alt="" height="24px" />
  </Navbar.Brand>
  <Navbar.Toggle aria-controls="basic-navbar-nav" />
  <Navbar.Collapse id="basic-navbar-nav" className="justify-content-end">
    <Nav>
      <Nav.Link className={styles.color} href="/">Home</Nav.Link>
      <Nav.Link className={styles.color} href="/">About Us</Nav.Link>
      <Nav.Link className={styles.color} href="/">Info</Nav.Link>
    </Nav>
  </Navbar.Collapse>
</Navbar>);

export default Index;
