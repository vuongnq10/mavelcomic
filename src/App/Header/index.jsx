import React from 'react';

import Navigation from './Navigation';
import styles from './styles.css';

const Index = () => (<div className={styles.home}>
  <div className="container">
    <Navigation />
    <h1 className={styles.slogan}>Save the world with Avenger team</h1>
    <h6 className={styles.description}>Winner winner chicken dinner</h6>
  </div>
</div>
);

export default Index;
