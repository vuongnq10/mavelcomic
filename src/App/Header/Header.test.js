import React from 'react';
import renderer from 'react-test-renderer';

import Header from './index';

it('Renders Header component', () => {
  const header = renderer
    .create(<Header />)
    .toJSON();

  expect(header).toMatchSnapshot();
});