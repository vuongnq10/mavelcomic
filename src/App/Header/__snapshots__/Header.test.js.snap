// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Header component 1`] = `
<div
  className={undefined}
>
  <div
    className="container"
  >
    <nav
      className="navbar navbar-expand-lg navbar-light"
    >
      <a
        className="navbar-brand"
        href="/"
      >
        <img
          alt=""
          height="24px"
          src="favicon.ico"
        />
      </a>
      <button
        aria-controls="basic-navbar-nav"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        onClick={[Function]}
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        aria-expanded={null}
        className="justify-content-end navbar-collapse collapse"
        id="basic-navbar-nav"
      >
        <div
          className="navbar-nav"
          onKeyDown={[Function]}
        >
          <a
            aria-controls={undefined}
            className="nav-link"
            data-rb-event-key="/"
            href="/"
            id={undefined}
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Home
          </a>
          <a
            aria-controls={undefined}
            className="nav-link"
            data-rb-event-key="/"
            href="/"
            id={undefined}
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            About Us
          </a>
          <a
            aria-controls={undefined}
            className="nav-link"
            data-rb-event-key="/"
            href="/"
            id={undefined}
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Info
          </a>
        </div>
      </div>
    </nav>
    <h1
      className={undefined}
    >
      Save the world with Avenger team
    </h1>
    <h6
      className={undefined}
    >
      Winner winner chicken dinner
    </h6>
  </div>
</div>
`;
